husky_control/config/control.yaml

```diff
@@ -30,13 +30,15 @@ husky_velocity_controller:
   # Whenever a min_* is unspecified, default to -max_*
   linear:
     x:
-      has_velocity_limits    : true
+      has_velocity_limits    : false
```

Comment:
not sure if its better to leave this true and up the max velocity, but I guess we can decide on that later